core/templates/components/oppia-angular-root.component.ts

```diff
@@ -196,6 +197,9 @@ export class OppiaAngularRootComponent implements AfterViewInit {
   }
 
   public ngAfterViewInit(): void {
+    if (!OppiaAngularRootComponent.contextService) {
```

Comment:
Add a TODO (create github issue): After angular migration completes, remove the usage of static requirements of context services. (Similar for rteHelperService and others)